README.md

```diff
@@ -1,264 +1,165 @@
-# traveltimePY: Travel Time Python SDK
+# Travel Time Python SDK
 
-traveltimePY is a Python SDK for Travel Time API (https://traveltime.com/).  
 Travel Time API helps users find locations by journey time rather than using ‘as the crow flies’ distance.  
 Time-based searching gives users more opportunities for personalisation and delivers a more relevant search.
 
-Dependencies:
-
-* requests
-
-## Installation
-
-```python
-    pip install traveltimepy
```

Comment:
will you create new package since you deleted this?

Reply:
I'll rework this part of readme

Reply:
I haven't understood yet how packaging in python is working